src/zap/zapinfo.cpp

```diff
@@ -3307,6 +3307,36 @@ unsigned ZapInfo::getClassSize(CORINFO_CLASS_HANDLE cls)
     return size;
 }
 
+unsigned ZapInfo::getHeapClassSize(CORINFO_CLASS_HANDLE cls)
+{
+#ifdef FEATURE_READYTORUN_COMPILER
+    if (IsReadyToRunCompilation())
+    {
+        _ASSERTE(!"getHeapClassSize");
+        ThrowHR(E_NOTIMPL);
+    }
+#endif
+
+    DWORD size = m_pEEJitInfo->getHeapClassSize(cls);
```

Comment:
Why not just `return m_pEEJitInfo->getHeapClassSize(cls);` ?

Reply:
Done.